package.json

```diff
@@ -5,22 +5,38 @@
   "main": "index.js",
```

Comment:
This comment is about `dist/index_bundle.js` file: it should be git ignored. Not source code? Should not be versioned.

Reply:
sorry, didn't understand that one...

Reply:
Over chat then